packages/PHPStanStaticTypeMapper/PHPStanStaticTypeMapper.php

```diff
@@ -12,7 +12,7 @@
 use Rector\Core\Exception\NotImplementedYetException;
 use Rector\PHPStanStaticTypeMapper\Contract\TypeMapperInterface;
 
-final class PHPStanStaticTypeMapper
+final class PHPStanStaticTypeMapper implements TypeMapperInterface
```

Comment:
not sure why this class exists. it is a kind of in-progress implementation which existed before this PR. 

I had to adjust it, so phpstan errors which get emitted because of the interface change get fixed.
the class is still mainly `NotImplementedYetException`.

maybe its just a leftover and can be completely deleted?